app/javascript/styles/mastodon/components.scss

```diff
@@ -6136,7 +6136,6 @@ a.status-card {
 .onboard-sliders {
   display: inline-block;
   max-width: 30px;
-  max-height: auto;
```

Comment:
`max-height: auto` probably was meant to be `max-height: none`, I'm not sure if this is needed.

Reply:
It's possible that `.onboard-sliders` may not be used anywhere anymore, it doesn't sound familiar to me.

Reply:
I have removed it. It seems to be a leftover from #1883.